app-shells/fzf/fzf-0.21.1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 eutils go-module
```

Comment:
*eutils* becomes redundant after your changes.

Reply:
Okay, gotta nuke it.